product-service/serverless.ts

```diff
@@ -0,0 +1,41 @@
+import type { AWS } from '@serverless/typescript';
+
+import hello from '@functions/hello';
+import getProductsList from '@functions/getProductsList';
+import getProductsById from '@functions/getProductsById';
+
+const serverlessConfiguration: AWS = {
+  service: 'product-service',
+  frameworkVersion: '3',
+  plugins: ['serverless-esbuild'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    stage: 'dev',
+    region: 'eu-west-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+  },
+  functions: { hello, getProductsList, getProductsById },
```

Comment:
Is `hello` function needed? Please remove it if it's not going to be used

Reply:
`hello` function don't interrupt the rest functionality. I need it for test proposes.

Reply:
no problem, thanks for clarification :)